applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -313,6 +352,21 @@ set_system_macros(Context) ->
             Context
     end.
 
+post(Context, ?CUSTOMER_UPDATE, ?MESSAGE) ->
+    case
+        whapps_util:amqp_pool_request(
```

Comment:
Please use `wh_amqp_worker:call/3` instead. I'd like to phase out the whapps_call amqp pool stuff as its not doing anything of note that `wh_amqp_worker` doesn't already do.